evennia/contrib/utils/database_backup/README.md

```diff
@@ -0,0 +1,69 @@
+# Database Backup Scheduler
+
+Contribution by helpme (2024)
+
+This module schedules backups in-game, which saves a copy of your database to your game's `server/backups` folder. Database backups are *not* automatically uploaded to any cloud service, it is left to you to decide what to do with them (i.e. pushed to git, uploaded to the ether, downloaded to a hard drive).
+
+Backups can take place at any time. Restoring the game world from backup takes place during downtime, as documented below.
+
+Currently, the sqlite3 (the evennia default) and postgresql databases are supported. Others are welcome to add more.
+
+## Installation
+
+This utility adds the `backup` command. Import the module into your commands and add it to your command set to make it available.
+
+In `mygame/commands/default_cmdsets.py`:
+
+```python
+...
+from evennia.contrib.utils.database_backup import DbCmdSet   # <---
+
+class CharacterCmdset(default_cmds.Character_CmdSet):
+    ...
+    def at_cmdset_creation(self):
+        ...
+        self.add(DbCmdSet)  # <---
+
+```
+
+Then `reload` to make the `backup` command available.
+
+## Permissions
+
+By default, the backup command is only available to those with Developer permissions and higher. You can change this by overriding the command and setting its locks from "cmd:pperm(Developer)" to the lock of your choice.
+
+## Settings Used
+
+This utility uses the settings.DATABASES dictionary.
+
+## Restoration
+
+Remember to `evennia stop` before restoring your db.
+
+### Restoring sqlite3 (.db3)
+
+* Copy the database backup you want to restore from back into the `server/` directory
+* Rename the database backup to `evennia.db3` if you have not modified `settings.DATABASES`, otherwise whatever name is in your `settings.DATABASES` dictionary.
+
+### Restoring postgres (.sql)
+
+* Prepare the following variables
+```
+export DB_USER=db_user from your settings.DATABASES
+export DB_NAME=db_name from your settings.DATABASES
+export BACKUP_FILE=the path to the backup file you are restoring from
+export PGPASSWORD=your db password
```

Comment:
This is hard to understand since it's all in \`\`\`. Is `export DB_USER=db_user from your settings.DATABASES` one command? Is `export DB_USER=db_user` the command you should run? Does 'export' here only mean you should find it and remember it? If you use \`\`\`, place the actual command alone (with comments between each block), or use clear `#`-markers to show what is a comment.